<launch>
    <node pkg="arx_x7_controller" type="x7_controller" name="right_arm" output="screen">
        <param name="arm_can_id" value="can1"/>
        <param name="arm_type" value="1"/>
        <remap from="/joint_information" to="/joint_information2"/>
        <remap from="/follow1_pos_back" to="/follow2_pos_back"/>
    </node>

    <!-- Right arm TF publisher -->
    <node pkg="arx_x7_controller" type="arm_state_publisher" name="right_arm_state_publisher" output="screen">
        <param name="arm_type" value="1"/>
    </node>
</launch>